webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+                'Incorrect arguments.\
+                 Either provide "csv files" or "ensembles and response_file".'
+            )
+        self.check_runs()
+        self.check_response_filters()
+        if response_ignore:
+            self.responsedf.drop(response_ignore, errors="ignore", axis=1, inplace=True)
+        if response_include:
+            self.responsedf.drop(
+                self.responsedf.columns.difference(
+                    ["REAL", "ENSEMBLE", *response_include, *list(response_filters.keys()),]
+                ),
+                errors="ignore",
+                axis=1,
+                inplace=True,
+            )
+        if parameter_ignore:
+            self.parameterdf.drop(parameter_ignore, axis=1, inplace=True)
+
+        self.theme = app.webviz_settings["theme"]
+        self.badge_style = {
+            "font-weight": "bold",
+            "fontSize": ".70em",
+            "borderRadius": "3px",
+            "backgroundColor": "#505050",
+            "color": "white",
+            "white-space": "pre-wrap",
+        }
+        self.tooltip_style = {
+            "fontSize": ".75em",
+            "backgroundColor": "#505050",
+            "color": "white",
+            "opacity": "85%",
+            "white-space": "pre-wrap",
+            "borderRadius": "5px",
+        }
+        self.set_callbacks(app)
+
+
+    @property
+    def tour_steps(self):
+        """ Adding a "Guided tour" functionality """
+        steps = [
+            {
+                "id": self.uuid("layout"),
+                "content": (
+                    "Dashboard displaying the results of a multiple "
+                    "regression of input parameters and a chosen response."
+                ),
+            },
+            {
+                "id": self.uuid("p-values-plot"),
+                "content": (
+                    "A plot showing the p-values for the parameters from the table ranked from most "
+                    "significant to least significant.  Red indicates that the p-value is "
+                    "significant, gray indicates that the p-value is not significant."
+                ),
+            },
+            {
+                "id": self.uuid("coefficient-plot"),
+                "content": (
+                    "A plot showing the sign of the parameters' coefficient values by arrows "
+                    "pointing up or down, illustrating a positive or a negative coefficient "
+                    "respectively. An arrow is red if the corresponding p-value is significant, "
+                    "that is, a p-value below 0.05. Arrows corresponding to p-values above this "
+                    "level of significance are shown in gray."
+                ),
+            },
+            {
+                "id": self.uuid("table"),
+                "content": (
+                    "A table showing the p-values for the best combination of "
+                    "parameters for a chosen response."
```

Comment:
Should be rewritten. It is not necessarily the best, only the one that we ended up with. Could be a local maxima.